tests/plot_test.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+
+import pytest
+import torch
+from torch.utils.tensorboard import SummaryWriter
+
+from sbi.inference import (
+    SNLE,
+    SNPE,
+    SNRE,
+    prepare_for_sbi,
+    simulate_for_sbi,
+)
+from sbi import utils
+from matplotlib.figure import Figure
+from matplotlib.axes import Axes
+
+
+def test_plot_summary(tmp_path):
```

Comment:
To avoid code repetition, you could parameterize the test as done e.g. [here](https://github.com/mackelab/sbi/blob/bc4d43bf60ec714790ef8baa4b08cc6e5b821e45/tests/inference_with_NaN_simulator_test.py#L48-L49)